tools/certify/commands/sign-ticket.cpp

```diff
@@ -113,8 +128,8 @@ int SignTicketCommand::execute()
     std::ofstream dest;
     dest.open(output.c_str(), std::ios::out | std::ios::binary);
 
-    vanetza::OutputArchive archive(dest);
-    vs::serialize(archive, certificate);
+    OutputArchive archive(dest);
+    serialize(archive, certificate);
```

Comment:
All changes in sign-ticket.cpp do not belong the added "show-cert" feature, so these should be moved to a separate commit.

Reply:
Indeed, I shouldn't be lazy. Extracted to #26.